descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Finance Funds",
+  "requires": [],
+  "provides": [
+    {
+      "id": "finances",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/funds",
+          "permissionsRequired": ["finance.funds.collection.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/finance/funds",
+          "permissionsRequired": ["finance.funds.item.post"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/funds/{id}",
+          "permissionsRequired": ["finance.funds.item.get"]
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/finance/funds/{id}",
+          "permissionsRequired": ["finance.funds.item.put"]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/finance/funds/{id}",
+          "permissionsRequired": ["finance.funds.item.delete"]
+        }
+      ]
+    },
+    {
+      "id": "_tenant",
+      "version": "1.0",
+      "interfaceType": "system",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/_/tenant"
+        },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/_/tenant"
+        }
+      ]
+    }
+  ],
+  "permissionSets": [
+    {
+      "permissionName": "finance.funds.collection.get",
+      "displayName": "Finances - get finance funds collection",
+      "description": "Get finance funds collection"
+    },
+    {
+      "permissionName": "finance.funds.item.get",
+      "displayName": "Finances - get individual finance funds from storage",
+      "description": "Get individual finance funds"
+    },
+    {
+      "permissionName": "finance.funds.item.post",
+      "displayName": "Finances - create finance funds",
+      "description": "Create finance funds"
+    },
+    {
+      "permissionName": "finance.funds.item.put",
+      "displayName": "Finances - modify finance funds",
+      "description": "Modify finance funds"
+    },
+    {
+      "permissionName": "finance.funds.item.delete",
+      "displayName": "Finances - delete finance funds",
+      "description": "Delete finance funds"
+    },
+    {
+      "permissionName": "finance.funds.all",
+      "displayName": "Finance Funds module - all permissions",
+      "description": "Entire set of permissions needed to use the Finance funds module",
+      "subPermissions": [
+        "finance.funds.collection.get",
+        "finance.funds.item.get",
+        "finance.funds.item.post",
+        "finance.funds.item.put",
+        "finance.funds.item.delete"
+      ],
+      "visible": false
+    }
+  ],
+  "metadata": {
+    "containerMemory": "256",
+    "databaseConnection": "true"
```

Comment:
According to [ModuleDescriptor guide](https://dev.folio.org/guides/module-descriptor/):
>`databaseConnection` – Whether this module utilises a database.

I believe that this property is not required here